src/quo/components/wallet/transaction_progress/view.cljs

```diff
@@ -0,0 +1,269 @@
+    :else                                                       :pending))
+
+(defn calculate-box-state-arbitrum-right
+  [state network]
+  (cond
+    (= state :error)                                    
+    :error
+    (and (= network :arbitrum) 
+         (= state :sending)) 
+    :confirmed
+    (or (= state :confirmed) 
+        (= state :finalising) 
+        (= state :finalized))  
+    :finalized
+    :else                                                       
+    :pending))
+
+(defn calculate-progressed-value
+  [state network]
+  (case state
+    :finalising (if (= network :arbitrum) 26 16)
+    :finalized  100
+    0))
+
+(defn progress-boxes-arbitrum
+  [state network bottom-large?]
+  [rn/view {:style (style/progress-box-container bottom-large?)}
+   [progress-box/view
+    {:state               (calculate-box-state-arbitrum-left state network)
+     :customization-color :blue}]
+   [progress-box/view
+    {:state               (calculate-box-state-arbitrum-right state network)
+     :full-width?         true
+     :progressed-value    (calculate-progressed-value state network)
+     :customization-color :blue}]])
+
+(defn text-internal
+  [title
+   {:keys [weight size accessibility-label color]
+    :or   {weight     :semi-bold
+           size       :paragraph-1}}]
+  [text/text
+   {:accessibility-label accessibility-label
+    :ellipsize-mode      :tail
+    :number-of-lines     1
+    :weight              weight
+    :size                size
+    :style               {:color color}}
+   title])
+
+(defn network-type-text
+  [network state]
+  (cond
+    (and (= state :sending) (= network :arbitrum))     "Confirmed on "
+    (or (= state :sending) (= state :pending))      "Pending on "
+    (or (= state :confirmed) (= state :finalising)) "Confirmed on "
+    (= state :finalized)                                    "Finalized on "
+    (= state :error)                                        "Failed on "))
+
+(defn text-steps
+  [network state epoch-number]
+  (cond
+    (and (= network :mainnet)
+         (not= state :finalized)
+         (not= state :error))       (str (if (< @lcounter 4)
+                                                   @lcounter
+                                                   "4")
+                                                 "/4")
+    (= state :finalized)            (str "Epoch " epoch-number)
+    (and (= network :mainnet)
+         (= state :error))          "0/4"
+    (and (= network :arbitrum)
+         (= state :confirmed))      "0/1"
+    (and (not= network :mainnet)
+         (= state :finalising))     "1/1"
+    (and (= network :arbitrum)
+         (= state :sending))        "0/1"
+    (not= network :mainnet)            "0/1"))
+
+(defn get-status-icon
+  [theme network state]
+  (cond
+    (and (= network :arbitrum)
+         (= state :sending))   [:i/positive-state
+                                        (colors/theme-colors colors/success-50
```

Comment:
please use `(colors/resolve-color :success theme)`
same for danger
 `(colors/resolve-color :danger theme)`

Reply:
Done